compiler/src/iree/compiler/Dialect/VM/Conversion/MathToVM/ConvertMathToVM.cpp

```diff
@@ -119,7 +119,10 @@ void populateMathToVMPatterns(MLIRContext *context,
       UnaryArithmeticOpConversion<math::TanhOp, IREE::VM::TanhF32Op,
                                   IREE::VM::TanhF64Op>,
       UnaryArithmeticOpConversion<math::ErfOp, IREE::VM::ErfF32Op,
-                                  IREE::VM::ErfF64Op>>(typeConverter, context);
+                                  IREE::VM::ErfF64Op>,
+      UnaryArithmeticOpConversion<math::RoundOp, IREE::VM::RoundF32Op,
```

Comment:
reorder to below floor

Reply:
Floor actually wasn't included. Added in the other operations but not sure how that factors into total ordering.

Reply:
Oh this list is weird? It's almost alphabetical? Maybe group the trig functions together and them order the rest alphabetically?

```
// Trigonometric functions.
patterns.insert< ...

// Other math functions.
patterns.insert< ...
```

or just add `round` in this PR, then in a follow-up add the other conversions that were missing

Reply:
Reordered to the same as op declarations.